packages/fxa-event-broker/config/index.ts

```diff
@@ -27,6 +27,12 @@ const conf = convict({
       doc: 'Refresh interval in seconds for fetching capability updates',
       env: 'CAPABILITY_REFRESH_INTERVAL',
       format: Number
+    },
+    requireCapabilities: {
+      default: true,
+      doc: 'Whether to fail to start if capabiltiies are not available from the auth server',
```

Comment:
Two small nits:

* Typo, "capabilities".

* I misread it the first couple of times as "whether to start if capabilities are not available..." , which is obviously the opposite of what's true. My eyes skipped over the all important "to fail" bit. Would it be clearer if it was just "whether to fail if capabilities are not available..." instead? (or possibly just poor reading skills on my part)

Reply:
Ah, thanks! Fixed the typo and re-worded the env var.